Index: src/main/kotlin/dev/newteam/newteam3/players/models/Persona.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package dev.newteam.newteam3.players.models\r\n\r\nclass Persona {\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/kotlin/dev/newteam/newteam3/players/models/Persona.kt b/src/main/kotlin/dev/newteam/newteam3/players/models/Persona.kt
--- a/src/main/kotlin/dev/newteam/newteam3/players/models/Persona.kt	(revision 557194f6ac89edd2cffaeaf260f83c7736514db9)
+++ b/src/main/kotlin/dev/newteam/newteam3/players/models/Persona.kt	(date 1747845970093)
@@ -1,4 +1,16 @@
 package dev.newteam.newteam3.players.models
 
-class Persona {
-}
\ No newline at end of file
+import java.time.LocalDateTime
+
+open class Persona(
+    val id: Int,
+    val nombre: String,
+    val apellido: String,
+    val fechaNacimiento: LocalDateTime,
+    val fechaIncorporacion: LocalDateTime,
+    val salario: Double,
+    val pais: String,
+    val imagen : String,
+    val createdAt: LocalDateTime,
+    val updateAt: LocalDateTime,
+)
\ No newline at end of file
Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 557194f6ac89edd2cffaeaf260f83c7736514db9)
+++ /dev/null	(revision 557194f6ac89edd2cffaeaf260f83c7736514db9)
@@ -1,8 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Editor-based HTTP Client requests
-/httpRequests/
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
Index: src/main/kotlin/dev/newteam/newteam3/players/models/Convocatoria.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/kotlin/dev/newteam/newteam3/players/models/Convocatoria.kt b/src/main/kotlin/dev/newteam/newteam3/players/models/Convocatoria.kt
new file mode 100644
--- /dev/null	(date 1747920548704)
+++ b/src/main/kotlin/dev/newteam/newteam3/players/models/Convocatoria.kt	(date 1747920548704)
@@ -0,0 +1,12 @@
+package dev.newteam.newteam3.players.models
+
+import java.time.LocalDateTime
+
+data class Convocatoria(
+    val id: Int,
+    val jornada: LocalDateTime,
+    val descripcion: String,
+    val jugadores: List<Jugador>,
+    val onceTitular: List<Jugador>,
+    val entrenador: Entrenador
+)
Index: .idea/gradle.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"GradleMigrationSettings\" migrationVersion=\"1\" />\r\n  <component name=\"GradleSettings\">\r\n    <option name=\"linkedExternalProjectsSettings\">\r\n      <GradleProjectSettings>\r\n        <option name=\"externalProjectPath\" value=\"$PROJECT_DIR$\" />\r\n        <option name=\"modules\">\r\n          <set>\r\n            <option value=\"$PROJECT_DIR$\" />\r\n          </set>\r\n        </option>\r\n      </GradleProjectSettings>\r\n    </option>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/gradle.xml b/.idea/gradle.xml
--- a/.idea/gradle.xml	(revision 557194f6ac89edd2cffaeaf260f83c7736514db9)
+++ b/.idea/gradle.xml	(date 1747845757465)
@@ -5,6 +5,7 @@
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
+        <option name="gradleJvm" value="#JAVA_HOME" />
         <option name="modules">
           <set>
             <option value="$PROJECT_DIR$" />
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ExternalStorageConfigurationManager\" enabled=\"true\" />\r\n  <component name=\"FrameworkDetectionExcludesConfiguration\">\r\n    <file type=\"web\" url=\"file://$PROJECT_DIR$\" />\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_21\" default=\"true\" project-jdk-name=\"liberica-21\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 557194f6ac89edd2cffaeaf260f83c7736514db9)
+++ b/.idea/misc.xml	(date 1747845757499)
@@ -4,7 +4,7 @@
   <component name="FrameworkDetectionExcludesConfiguration">
     <file type="web" url="file://$PROJECT_DIR$" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_21" default="true" project-jdk-name="liberica-21" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_21" default="true" project-jdk-name="liberica-full-21" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
Index: .idea/material_theme_project_new.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"MaterialThemeProjectNewConfig\">\r\n    <option name=\"metadata\">\r\n      <MTProjectMetadataState>\r\n        <option name=\"migrated\" value=\"true\" />\r\n        <option name=\"pristineConfig\" value=\"false\" />\r\n        <option name=\"userId\" value=\"-3369697b:192aff62f17:-7ffe\" />\r\n      </MTProjectMetadataState>\r\n    </option>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/material_theme_project_new.xml b/.idea/material_theme_project_new.xml
--- a/.idea/material_theme_project_new.xml	(revision 557194f6ac89edd2cffaeaf260f83c7736514db9)
+++ b/.idea/material_theme_project_new.xml	(date 1747845757514)
@@ -5,7 +5,7 @@
       <MTProjectMetadataState>
         <option name="migrated" value="true" />
         <option name="pristineConfig" value="false" />
-        <option name="userId" value="-3369697b:192aff62f17:-7ffe" />
+        <option name="userId" value="120f90f5:192aff69e6c:-7ffe" />
       </MTProjectMetadataState>
     </option>
   </component>
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"VcsDirectoryMappings\">\r\n    <mapping directory=\"$PROJECT_DIR$\" vcs=\"Git\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
--- a/.idea/vcs.xml	(revision 557194f6ac89edd2cffaeaf260f83c7736514db9)
+++ b/.idea/vcs.xml	(date 1747845757529)
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="" vcs="Git" />
   </component>
 </project>
\ No newline at end of file
Index: src/main/kotlin/dev/newteam/newteam3/players/models/User.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/kotlin/dev/newteam/newteam3/players/models/User.kt b/src/main/kotlin/dev/newteam/newteam3/players/models/User.kt
new file mode 100644
--- /dev/null	(date 1747920317725)
+++ b/src/main/kotlin/dev/newteam/newteam3/players/models/User.kt	(date 1747920317725)
@@ -0,0 +1,12 @@
+package dev.newteam.newteam3.players.models
+
+class User(
+    val nombreDeUser: String,
+    val passwordHash: String,
+    val rol: Rol
+) {
+    enum class Rol {
+        ADMIN, USER
+    }
+
+}
\ No newline at end of file
Index: src/main/kotlin/dev/newteam/newteam3/players/models/Entrenador.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/kotlin/dev/newteam/newteam3/players/models/Entrenador.kt b/src/main/kotlin/dev/newteam/newteam3/players/models/Entrenador.kt
new file mode 100644
--- /dev/null	(date 1747909233537)
+++ b/src/main/kotlin/dev/newteam/newteam3/players/models/Entrenador.kt	(date 1747909233537)
@@ -0,0 +1,35 @@
+package dev.newteam.newteam3.players.models
+
+import java.time.LocalDateTime
+
+class Entrenador(
+    id: Int,
+    nombre: String,
+    apellido: String,
+    fechaNacimiento: LocalDateTime,
+    fechaIncorporacion : LocalDateTime,
+    salario : Double,
+    pais: String,
+    imagen: String,
+    createdAt: LocalDateTime,
+    updateAt: LocalDateTime,
+    val especializacion: Especializacion
+): Persona(id,
+    nombre,
+    apellido,
+    fechaNacimiento,
+    fechaIncorporacion,
+    salario,
+    pais,
+    imagen,
+    createdAt,
+    updateAt
+) {
+    enum class Especializacion {
+        PRINCIPAL, ASISTENTE, PORTEROS
+    }
+
+    override fun toString(): String {
+        return ("(Entrenador(id=$id, nombre=$nombre, apellido=$apellido, fechaNacimiento=$fechaNacimiento, fechaIncorporacion= $fechaIncorporacion, salario= $salario, pais=$pais, especializacio= $especializacion)")
+    }
+}
\ No newline at end of file
Index: src/main/kotlin/dev/newteam/newteam3/players/models/Jugador.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/kotlin/dev/newteam/newteam3/players/models/Jugador.kt b/src/main/kotlin/dev/newteam/newteam3/players/models/Jugador.kt
new file mode 100644
--- /dev/null	(date 1747908719738)
+++ b/src/main/kotlin/dev/newteam/newteam3/players/models/Jugador.kt	(date 1747908719738)
@@ -0,0 +1,42 @@
+package dev.newteam.newteam3.players.models
+
+import java.time.LocalDateTime
+
+class Jugador(
+    id: Int,
+    nombre: String,
+    apellido: String,
+    fechaNacimiento: LocalDateTime,
+    fechaIncorporacion : LocalDateTime,
+    salario : Double,
+    pais: String,
+    imagen: String,
+    createdAt: LocalDateTime,
+    updateAt: LocalDateTime,
+    val posicion: Posicion,
+    val dorsal: Int,
+    val altura: Double,
+    val peso: Double,
+    val goles: Int,
+    val partidosJugados: Int,
+    val minutosJugados: Int
+): Persona(id,
+    nombre,
+    apellido,
+    fechaNacimiento,
+    fechaIncorporacion,
+    salario,
+    pais,
+    imagen,
+    createdAt,
+    updateAt
+) {
+
+    enum class Posicion {
+        PORTERO, DEFENSA, MEDIOCENTRO, DELANTERO
+    }
+
+    override fun toString(): String {
+        return ("Jugador(id=$id, nombre=$nombre, apellido=$apellido, fechaNacimiento=$fechaNacimiento, fechaIncorporacion=$fechaIncorporacion, salario=$salario, pais=$pais, posicion=$posicion, dorsal=$dorsal, altura=$altura, peso=$peso, goles=$goles, partidosJugados=$partidosJugados, minutosJugados=$minutosJugados)" )
+    }
+}
\ No newline at end of file
